test/testMdastToVDOM.js

```diff
@@ -33,6 +33,9 @@ const assertTransformerYieldsDocument = (transformer, expected) => {
     new JSDOM(expected).window.document,
   );
 
+  // Reset the transformer between the 2 calls to avoid leakages in the tests
+  transformer.reset();
```

Comment:
Need to reset the transformer between transformations to avoid leakages in the tests